demisto_sdk/commands/validate/validators/PB_validators/PB106_is_playbook_using_an_instance.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from typing import ClassVar, Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.base_playbook import TaskConfig
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsPlayBookUsingAnInstanceValidator(BaseValidator[ContentTypes]):
+    invalid_tasks: ClassVar[dict[str, list[TaskConfig]]] = {}
+    error_code = "PB106"
+    description = "Validate whether the playbook does not use an instance. If the Playbook use an instance it is not valid."
+    rationale = "If the playbook uses a specific instance it can leads to errors because not all the users have the same instance"
```

Comment:
```suggestion
    rationale = "If the playbook uses a specific instance it can leads to errors because not all the users have the same instance."
```